tests/integration/data_quality_test.py

```diff
@@ -54,11 +54,11 @@ def test_lost_facts_pct(extracted, request):
         # We have unallocated data for Form 6 for some reason.
         total_threshold = 0.9
         per_filing_threshold = 0.8
-        # Assert that this is < 0.95 so we remember to fix this test once we
+        # Assert that this is < 0.96 so we remember to fix this test once we
```

Comment:
This number went up a little bit, but not a lot, and we didn't actively try to fix anything here, so I just bumped the "we still need to fix it" threshold.